src/main/kotlin/org/malachite/estella/task/api/TaskController.kt

```diff
@@ -81,9 +81,9 @@ class TaskController(
     fun startTask(
             @RequestParam("taskStage") taskStageUuid: String,
             @RequestParam("task") taskId: Int
-    ) {
-        taskStageService.startTask(taskStageUuid, taskId)
-    }
+    ) = taskStageService.startTask(taskStageUuid, taskId)
```

Comment:
No nie którzy nie przepadają, ale ja lubię mieć typ zwracany bo jak coś się zmieni to możesz zauważyć, że typy się wywaliły. Nie chodzi mi o ten case, a bardziej o konwencję.

Reply:
w tym przypadku to byłoby `ResponseEntity<Any>` a wiem, że @Davenury kiedyś wspominał że to jest trochę nic nieznaczący bełkot XD